tools/detect_nonexistent_testdata

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+FILES="${@}"
+success=1
+for FILE in $FILES; do
```

Comment:
/FILE/file/ following variables convention

Reply:
more like `s/success/SUCCESS/` for uniformity, I think in bourne shell we are free to mix: https://www.shellscript.sh/variables1.html ?

Reply:
I cannot make these kind of aesthetic changes as I shamelessly stole Joaquin's code.  I have some dignity xD

Reply:
I am leaving this here. https://unix.stackexchange.com/a/42849/115625
We have to decide in some point if we care about that kind of things or not. Confuses everyone i think

Reply:
I think we care about that kind of things in Perl, which is the main language of the repo and most of the code follows some style, but with scripting... @b10n1k ,to be honest I'm not sure. I also borrowed some part of the code from someone else's.